### PR TITLE
Card logos in upper right corner (online/onsite)

### PR DESCRIPTION
**Pull request by NelsonTheBrave**
This is how it looks, good? Let me know if you think the color or size should be different. 
OBS, chose a slightly different logo for onsite than the one in Trello because it was the closest I could find using font awesome
image

### CODE REVIEW

File: challenges.js [hunk 1]
```diff
@@ -13,6 +12,13 @@ class Challenge {
     challengeCard.id = this.data.id;
     challengeCard.classList.add('challenges-container__challenge');
     challengeCard.classList.add(this.data.type);
+    if (this.data.type === 'online') {
+      challengeCard.innerHTML = '<i class="fa-solid fa-globe"></i>';
+    } else {
+      challengeCard.innerHTML = '<i class="fa-solid fa-building-user"></i>';
+    }
+
+    console.log(challengeCard.innerHTML);
```
Comment (JrSchoolAccount) [inline challenges.js:21]: Don't forget to remove the below, if it only was there for testing. 😊

**Review by JrSchoolAccount — COMMENTED** [review]
Looks good 👍
Keep up the good work.

File: challenges.js (hunk 1 above)
Comment (NelsonTheBrave) [reply challenges.js:21]: Yes thank you, I will re-commit and remove it!

Comment (NelsonTheBrave) [reply challenges.js]: Done!

**Review by JrSchoolAccount — APPROVED** [review]
👍🏼

**Comment by MickeMarten** [comment]
Merge!

**Pull request merged**